assets/app/widgets/suse-modal/suse-modal.directive.jade

```diff
@@ -0,0 +1,9 @@
+.modal-content
+    .modal-header
+        button.close(type='button', ng-click='dismiss()', aria-label='Close')
+            span(aria-hidden='true') ×
+        h4#suseModalLabel.modal-title {{ error.title }}
```

Comment:
`suse-modal` sounds generic enough to be also used by a login form, not just errors.
As a suse directive, it should be generic enough to be reused in other pages/projects/contexts, and there is my concern